src/main/java/sim/coffee/QueueType.java

```diff
@@ -0,0 +1,6 @@
+package sim.coffee;
+
+public enum QueueType {
+	CUSTOMER,
+	KITCHEN;
```

Comment:
Are we still working with a Priority Queue? 😅

Reply:
This class is used to make a correct entry in the log (with Customer queue encompassing both ordinary/ priority) but might need deleting anyway depending on how we decide to restructure the Logger class.